dissect/target/loaders/remote.py

```diff
@@ -15,10 +15,20 @@
 from dissect.target.containers.raw import RawContainer
 from dissect.target.exceptions import LoaderError
 from dissect.target.loader import Loader
+from dissect.target.plugin import arg
 from dissect.target.target import Target
 
 log = logging.getLogger(__name__)
 
+option_client_key = "key"
```

Comment:
I'm not a big fan of making a constant out of every single string/option. These values won't be used outside of this file, can't we just put the strings in the code? It now requires a lot of scrolling up and down to check the actual values with (IMO) little added benefit of putting them here.

Reply:
These files are small enough and their uses close enough together that I'm personally not concerned for that.

Reply:
ok